tests/core/tools.js

```diff
@@ -855,6 +855,47 @@
 			assert.areSame( escapedSelector, '\\30 ', 'only-one-number selector' );
 		},
 
+		'test escapeCss - escaped colon in the css selector': function() {
```

Comment:
We usually put reference to an issue which causes us to add new tests:
```suggestion
		// (#681)
		'test escapeCss - escaped colon in the css selector': function() {
```

Please add such comment to all new tests.